### PR TITLE
test(e2e): add webpack support

### PR DESCRIPTION
**Pull request by sis0k0**

### CODE REVIEW

**Comment by sis0k0** [comment]
test

**Comment by sis0k0** [comment]
test

**Comment by sis0k0** [comment]
test

**Comment by sis0k0** [comment]
test

**Comment by sis0k0** [comment]
test

**Comment by sis0k0** [comment]
test

**Comment by sis0k0** [comment]
test

File: e2e/renderer/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "nativescript": {
     "id": "org.nativescript.renderer",
     "tns-android": {
-      "version": "next"
+      "version": "3.4.1-2018-02-06-02"
```
Comment (SvetoslavTsenov) [inline e2e/renderer/package.json]: next?

**Comment by SvetoslavTsenov** [comment]
👍

**Pull request merged**